test_basics.py

```diff
@@ -16,3 +16,8 @@ def test_admin_response(client):
 @mark.django_db()
 def test_database():
     get_user_model().objects.all().exists()
+
+@mark.django_db()
+def test_heartbeat_response(client):
+    response = client.get("/heartbeat")
```

Comment:
I'm not sure, but if you update the path url you might need to add a trailing slash here. Let's just see after the CI kicks off.